### PR TITLE
RRTMGP in fv3atm (#26 - based on latest code)

### PR DESCRIPTION
**Pull request by climbfuji**
Original PR: https://github.com/NCAR/fv3atm/pull/26

In addition: bugfixes and build system updates required for RRTMGP (see https://github.com/NCAR/ccpp-framework/pull/270 for more information on the build system update).

The PR also includes an update of the NCAR dtc/develop branch from the NOAA-EMC develop branch.

Associated PRs:

https://github.com/NCAR/ccpp-framework/pull/270
https://github.com/NCAR/ccpp-physics/pull/411 (includes https://github.com/NCAR/ccpp-physics/pull/413; the latter supersedes https://github.com/NCAR/ccpp-physics/pull/400)
https://github.com/NCAR/fv3atm/pull/32 (includes https://github.com/NCAR/fv3atm/pull/26)
https://github.com/NCAR/ufs-weather-model/pull/29

For detailed information on the RRTMGP code to be committed, see https://github.com/NCAR/ccpp-physics/pull/400.

For regression testing information, see https://github.com/NCAR/ufs-weather-model/pull/29.

### CODE REVIEW

File: ccpp/suites/suite_FV3_GFS_2017_RRTMGP.xml [hunk 1]
```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="FV3_GFS_2017_RRTMGP" lib="ccppphys" ver="3">
```
Comment (grantfirl) [inline ccpp/suites/suite_FV3_GFS_2017_RRTMGP.xml]: ver="4"

Comment (grantfirl) [reply ccpp/suites/suite_FV3_GFS_2017_RRTMGP.xml]: Also, the SDF in the SCM was based on an older version of the GFS suite. Should the RRTMGP suite be updated to be based on GFS_v15p2 or GFS_v16beta?

Comment (climbfuji) [reply ccpp/suites/suite_FV3_GFS_2017_RRTMGP.xml]: No, the version is still 3 in ufs-weather-model (except for the public release code). We will try to add or replace the RRTMGP test with a GFSv15p2 setup in the future. Until now, Dustin has been testing with the old control (FV3_GFS_2017) setup.

Comment (grantfirl) [reply ccpp/suites/suite_FV3_GFS_2017_RRTMGP.xml]: OK. For the SCM PR, I have the SDF using RRTMGP based on GFS_v15p2. I can revert that change if necessary.

Comment (climbfuji) [reply ccpp/suites/suite_FV3_GFS_2017_RRTMGP.xml]: No, I think that's totally fine and a good test that it works in principle with GFSv15p2 (with a different MP scheme and more hydrometeors). It just hasn't been tested in FV3. Maybe we do that as part of the radiation tendencies cleanup (and other RRTMGP cleanup)? Would be nice to have it before sending the code to EMC.

**Pull request merged**